.github/workflows/changelog.yml

```diff
@@ -0,0 +1,10 @@
+name: Check changelog file included
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
```

Comment:
Thanks for opening the PR @jjmerchante 

Coming from the comment https://github.com/chaoss/grimoirelab-toolkit/pull/46#discussion_r778047400

Do you think we can add a similar rule for perceval too?

```
on:
  pull_request:
    paths:
      - 'perceval/'
```

This would avoid the workflow running for all the PRs. Instead, it will be triggered only when there is a change in the source code (perceval folder).

I'm not sure about the `branches` rule. It should be helpful if we are having a developing branch similar to sortinghat and bestiary. Do you have any other advantage that I'm missing?